.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: Build struct2tensor wheels
+
+on:
+  pull_request:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build package
+        id: build-package
+        uses: ./.github/reusable-build
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Retrieve wheels
+        uses: actions/download-artifact@v4
+        with:
+          merge-multiple: true
+          path: wheels
```

Comment:
Do we need to retrieve wheels in this job as it should be present already built from previous step?

Reply:
Right - this is an action that runs in the same job. Thanks for this!